workspace/TS100/configuration.h

```diff
@@ -87,7 +88,7 @@
    */
   #define OP_AMP_Rf               750*1000  // 750  Kilo-ohms -> From schematic, R1
   #define OP_AMP_Rin              2370      // 2.37 Kilo-ohms -> From schematic, R2
-  #define TIP_GAIN                405
```

Comment:
Please dont just comment it out. If you have replace its usage then remove the `#ifdef` entirely